packaging/windows/WIXInstallers/KeybaseApps/KeybaseApps.wxs

```diff
@@ -30,8 +30,8 @@
     <InstallExecuteSequence>
       <InstallValidate Suppress="yes">FAKE_PROPERTY</InstallValidate>
       <Custom Action="StopUpdater" Before="InstallValidate"/>
+      <Custom Action="StopMainApp" Before="StopGUI"/>
```

Comment:
In some cases on OSX, the GUI can start the App.  I'm guessing not here, but thought I'd double check